CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Optimizely Ruby SDK Changelog
 
 ## 4.0.0
-August 1st, 2022
+August 3st, 2022
```

Comment:
No need to add a date and version here. Version and date will be added once we are ready to release. This should go in unreleased. It should be 

## Unreleased

###  Changes
* Breaking change: Changed official supported versions of Ruby to 2.7, 3.0 and 3.1